tests/phpunit/elementor/modules/lazyload/test-module.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Elementor\Testing\Modules\LazyLoad;
+
+use ElementorEditorTesting\Elementor_Test_Base;
+use Elementor\Modules\LazyLoad\Module as LazyLoad;
+
+class Elementor_Test_LazyLoad extends Elementor_Test_Base {
+
+	public function test_reduce_background_image_size() {
+		$image_id = $this->factory()->attachment->create_upload_object(dirname( __DIR__, 3 ) . '/resources/mock-image.png');
+		$image_url = wp_get_attachment_url( $image_id );
+		$reflection = new \ReflectionClass( LazyLoad::class );
+		$method = $reflection->getMethod( 'reduce_background_image_size' );
+		$method->setAccessible( true );
```

Comment:
bruh....
you shouldn't do that....

Reply:
Why? What would you suggest me to do instead?